NGitLab/Impl/MergeRequestClient.cs

```diff
@@ -156,6 +158,11 @@ public GitLabCollectionResponse<MergeRequestVersion> GetVersionsAsync(int mergeR
             return _api.Get().GetAllAsync<MergeRequestVersion>(_projectPath + "/merge_requests/" + mergeRequestIid.ToString(CultureInfo.InvariantCulture) + "/versions");
         }
 
+        public Task<TimeStats> TimeStatsAsync(int projectId, int mergeRequestIid, CancellationToken cancellationToken = default)
```

Comment:
It seems that all other methods in MergeRequestClient already knows the projectId.

Could you remove the param and use the existing `_projectPath` ?

Reply:
Done